fact_extractor/install/pre_install.sh

```diff
@@ -4,7 +4,9 @@ echo "Install Pre-Install Requirements"
 (apt-get update && apt-get install sudo) || true
 
 sudo apt-get update
-sudo apt-get -y install python-pip python3-pip python3-wheel python3-setuptools git apt-transport-https ca-certificates curl software-properties-common wget libmagic-dev
+sudo apt-get -y install python-pip
```

Comment:
You might want to append `|| true` to the line to make sure the script does not end prematurely

Reply:
I added that for clarity, though in my experience it doesn't seem to interrupt the script even if the package doesn't exist (on Debian 11 and Ubuntu 20.04)